docs/upgrade/12_to_13/postgresql.sql

```diff
@@ -0,0 +1,2 @@
+CREATE INDEX IF NOT EXISTS IX_oc_job_argument_id ON oc_job_argument (id);
+CREATE INDEX IF NOT EXISTS IX_oc_job_argument_id_argument ON oc_job_argument (id, argument);
```

Comment:
This is not going to work. I receive the following error message from our PostgreSQL:

```
index row requires 10032 bytes, maximum size is 8191.
```
This seems to be a hard limit on PostgreSQL.

Reply:
Others seem to have stumbled across this before: https://postgrespro.com/list/thread-id/1506742